PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera/FollowCamera.rbxmx

```diff
@@ -148,9 +149,18 @@ local function CreateFollowCamera()
 						end
 					elseif not (isInFirstPerson or userRecentlyPannedCamera) and not VRService.VREnabled then
 						local lastVec = -(self.LastCameraTransform.p - subjectPosition)
+						
 						local y = findAngleBetweenXZVectors(lastVec, self:GetCameraLook())
+						
+						-- This cutoff is to decide if humanoids angle of movement,
+						-- relative to the camera's look vector, is enough that
+						-- we wan the camera to be following them. The point is to provide 
```

Comment:
looks like there are a couple typos in this description, perhaps it is in 'want'! :P